src/factions/hearthkynSalvager.ts

```diff
@@ -191,6 +192,116 @@ const petitionTheGuild: Requisition = {
   rule: 'Purchase this Requisition before a game, if your kill team is currently conducting a Spec Op. Randomly determine one Recon or Security Tac Op. Until that Spec Op ends, keep a tally of victory points you score from that Tac Op. When you add the fifth mark to the tally, you can add one item of rare equipment to your stash.',
 }
 
+const hearthkynTheyn: Operatives = {
+  type: 'HEARTHKYN THEYN',
+  specialisms: [],
```

Comment:
they should have these. These are the ones thay can pick from. so each operative will have a few. for exmple the Theyn would be 'Combat, Staunch, Marksman'